### PR TITLE
Update dependencies and adapt to tonic best practice

### PR DESCRIPTION
**Pull request by bachp**
This is a refresh to bring the dependencies up to date.

### CODE REVIEW

File: plugin/Cargo.toml [hunk 1]
```diff
@@ -5,8 +5,9 @@ authors = ["hasheddan <georgedanielmangum@gmail.com>"]
...
 
 [dependencies]
-k8s-deviceplugin = "0.1.0"
-tonic = "0.4"
-prost = "0.7"
+k8s-deviceplugin = { path = "../k8s-deviceplugin" }
+tonic = "0.6"
+prost = "0.9"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
+tokio-stream = "0.1"
```
Comment (hasheddan) [inline plugin/Cargo.toml]: Is this being utilized anywhere?

**Review by hasheddan — COMMENTED** [review]
@bachp thanks for the updates and apologies for the delayed response! One question below but otherwise LGTM :)

File: plugin/Cargo.toml (hunk 1 above)
Comment (bachp) [reply plugin/Cargo.toml]: No it's only used indirectly. I removed it.

Comment (hasheddan) [reply plugin/Cargo.toml]: Thanks!

**Comment by bachp** [comment]
@hasheddan Will you do a new release on crates.io?

**Comment by hasheddan** [comment]
@bachp I would be happy to 👍🏻 Are you currently consuming this crate in a project?

**Comment by bachp** [comment]
@hasheddan Yes I'm using it on a device plugin prototype. Currently I have a copy but if a new release was done I can switch to it. 

The second issue currently preventing rust from being used out of the box for this is https://github.com/hyperium/h2/pull/612

**Comment by hasheddan** [comment]
@bachp awesome, I'll try to get a release out today 👍🏻

**Comment by hasheddan** [comment]
@bachp release is out https://crates.io/crates/k8s-deviceplugin/0.2.0

**Pull request merged**